ups/eupspkg.cfg.sh

```diff
@@ -1,7 +1,56 @@
-export SCONSFLAGS=$SCONSFLAGS" USE_UNKNOWN_VARS=true TMV_DIR="$TMV_DIR" PREFIX="$PREFIX" PYPREFIX="$PREFIX"/lib/python EXTRA_LIB_PATH="$TMV_DIR"/lib EXTRA_INCLUDE_PATH="$TMV_DIR"/include"
+export SCONSFLAGS=$SCONSFLAGS" USE_UNKNOWN_VARS=true TMV_DIR="$TMV_DIR\
+" PREFIX="$PREFIX" PYPREFIX="$PREFIX"/lib/python EXTRA_LIB_PATH="$TMV_DIR\
+"/lib EXTRA_INCLUDE_PATH="$TMV_DIR"/include"
 
 pathToPython=$(which python)
-export DYLD_FALLBACK_LIBRARY_PATH="${pathToPython%bin/python}/lib"
+pathToPythonLib="${pathToPython%bin/python}/lib"
```

Comment:
This is not a very robust way to find the python library location. In a virtualenv the library won't be there. I'm not sure if this is any better though:
```shell
% python -c 'import sysconfig; print(sysconfig.get_config_vars("LIBDIR")[0])'
```